exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+
+
+def read_file_move_to_dict(file_path):
+    """Read file and move atm information to dictionary.
+
+    Args:
+        file_path (string): path to atm information file.
```

Comment:
in the brackets, you need to specify the type of the attribute.
in python, the type of string is `str`, not `string`.

```suggestion
        file_path (str): path to atm information file.
```